docs/tutorial/working-with-pytrees.md

```diff
@@ -0,0 +1,3 @@
+# Working with pytrees
```

Comment:
PyTrees, pytrees -- if the latter, we should make it universal unless it already is

Reply:
Done